test/__snapshots__/express.test.ts.snap

```diff
@@ -115,6 +154,75 @@ exports[`mapping Express.js requests 1`] = `
       "parent_id": 5,
       "thread_id": 0,
     },
+    {
+      "event": "call",
+      "http_server_request": {
+        "headers": {
+          "host": "localhost:27627",
+        },
+        "path_info": "/api/foo",
```

Comment:
I'm noticing this doesn't provide `normalized_path`. When is normalized path available? I see it above.

Reply:
It's there in `eventUpdates`.